build-tests/heft-webpack-basic-test/webpack.config.js

```diff
@@ -1,17 +1,55 @@
 'use strict';
 
 const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 
-module.exports = {
-  mode: 'development',
-  resolve: {
-    extensions: ['.js', '.jsx', '.json']
+/** example config, each parameter has options for different build runtime environments */
+const CONFIG = {
+  reactUrl: {
+    dev: 'https://cdnjs.cloudflare.com/ajax/libs/react/16.4.2/umd/react.development.js',
+    production: 'https://cdnjs.cloudflare.com/ajax/libs/react/16.4.2/umd/react.production.min.js'
   },
-  entry: {
-    'heft-test': path.join(__dirname, 'lib', 'index.js')
-  },
-  output: {
-    path: path.join(__dirname, 'dist'),
-    filename: '[name]_[contenthash].js'
+
+  reactDomUrl: {
+    dev: 'https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.4.2/umd/react-dom.development.js',
+    production: 'https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.4.2/umd/react-dom.production.min.js'
   }
```

Comment:
It might be a good test to just bundle react and react-dom instead of externalizing them like we do in the TSDoc Playground.

Reply:
Agreed, the reason for externalizing them was due to a legal issue that doesn't apply here.